helm/distributed-idol/templates/dah/deployment.yaml

```diff
@@ -41,59 +33,25 @@ spec:
         - sh
         - "-c"
         - |
-{{ tpl (.Files.Get "resources/dah_init.sh") . | indent 10 }}
+{{ tpl ($root.Files.Get "resources/dah_init.sh") $root | indent 10 }}
         volumeMounts:
         - name: config-map
           mountPath: /mnt/config-map
         - name: config-volume
           mountPath: /mnt/config/idol
-      imagePullSecrets:
-        {{- range .Values.global.imagePullSecrets }}
-        - name: {{ . }}
-        {{- end }}
-      containers:
-      - name: {{ $component.name }}
-        image: {{ include "idol-library.idolImage" (dict "root" . "idolImage" $component.idolImage )}}
-        imagePullPolicy: IfNotPresent
-        command: ["/bin/sh"]
-        args: ["-c", "cd /dah && command cp -f /etc/config/idol/dah.cfg dah.cfg && ./run_idol.sh"]
-        lifecycle:
-          preStop:
-            exec:
-              command:
-              - /bin/bash
-              - "-c"
-              - |
-{{ tpl (.Files.Get "resources/dah_preStop.sh") . | indent 16 }}
-        volumeMounts:
-        - name: config-volume
-          mountPath: /etc/config/idol
-        {{- range $component.additionalVolumeMounts }}
-        - {{ . | toYaml | nindent 10 }}
-        {{- end }}
-        env:
-        - name: IDOL_COMPONENT_CFG
-          # Override default in newer IDOL containers
-          value: "/dah/dah.cfg"
-        livenessProbe:
-          httpGet:
-            path: /a=getpid
-            port: {{ $component.aciPort | int }}
-{{- template "idol-library.standardLivenessProbe" $component.livenessProbe }}
-        ports:
-        - containerPort: {{ $component.aciPort | int }}
-          name: aci-port
-          protocol: TCP
-        - containerPort: {{ $component.servicePort | int }}
-          name: service-port
-          protocol: TCP
       volumes:
       - name: config-volume
         emptyDir: {}
       - name: config-map
         configMap:
           name: {{ default (printf "%s-default-cfg" $component.name) $component.existingConfigMap }}
```

Comment:
This wants to use the volumes arg from `idol-library.aciserver.deployment.base.v1

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/80/commits/4e2a514f7355fd50081eeb8aecb793dbc8676a5f